sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -145,11 +149,15 @@ def from_connection_string(
                 :caption: Create a new instance of the ServiceBusClient from connection string.
 
         """
-        host, policy, key, entity_in_conn_str = _parse_conn_str(conn_str)
+        host, policy, key, entity_in_conn_str, token, token_expiry = _parse_conn_str(conn_str)
+        if token and token_expiry:
+            credential = ServiceBusSASTokenCredential(token, token_expiry)
+        elif policy and key:
+            credential = ServiceBusSharedKeyCredential(policy, key)
```

Comment:
same priority question to the `_create_credential_from_connection_string_parameters` one

Reply:
See above